English_transcripts/school/Epic Quest/sharedAssets/P_C_M_ENG_L3_AU_numline_dotb_15.txt

```diff
@@ -0,0 +1 @@
+. B is right after 15. what number is shown by . B?
```

Comment:
Are the full stops intentional?

Reply:
These full stops are intentional as per the English audio.